chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+    return text;
+  }
+
+  /**
+   * Make displayable URI
+   */
+  uriForDisplay(uri) {
+    const uri_display_trigger_length_k = 20;
+    const uri_length_max_k = 40;
+    let uri_display = uri || '';
+    if (uri_display.length > uri_display_trigger_length_k) {
+      let re = RegExp('^\\w+://([\\w./_-]+).*?([\\w._-]*)$');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

're' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)